### PR TITLE
EES-130 Fix table tool locations collapsible list

### PR DESCRIPTION
**Pull request by markhiveit**

### CODE REVIEW

File: src/explore-education-statistics-common/src/modules/table-tool/components/LocationFiltersForm.tsx [hunk 1]
```diff
@@ -170,10 +170,10 @@ const LocationFiltersForm = (props: Props & InjectedWizardProps) => {
                     shouldCollapse
                   >
                     {sortBy(levelOptions, ['label']).map(level => (
-                      <React.Fragment key={level.value}>
+                      <div key={level.value}>
```
Comment (ntsim) [inline src/explore-education-statistics-common/src/modules/table-tool/components/LocationFiltersForm.tsx:173]: Think these should be `li`s. Think this current change would result in non-semantic markup that looks like:

```html
<ul>
   <div>Location 1</div>
   <div>Location 2</div>
</ul>
```

File: src/explore-education-statistics-common/src/modules/table-tool/components/LocationFiltersForm.tsx [hunk 2]
```diff
@@ -170,10 +170,10 @@ const LocationFiltersForm = (props: Props & InjectedWizardProps) => {
...
                         {level.label}
                         <br />
```
Comment (ntsim) [inline src/explore-education-statistics-common/src/modules/table-tool/components/LocationFiltersForm.tsx:175]: I think if we use `li`s we don't need to use `<br />`s anymore.

File: src/explore-education-statistics-common/src/modules/table-tool/components/LocationFiltersForm.tsx (hunk 1 above)
Comment (markhiveit) [reply src/explore-education-statistics-common/src/modules/table-tool/components/LocationFiltersForm.tsx:173]: You are right -- in fact I actually made that change at one point

File: src/explore-education-statistics-common/src/modules/table-tool/components/LocationFiltersForm.tsx [hunk 3]
```diff
@@ -170,10 +170,7 @@ const LocationFiltersForm = (props: Props & InjectedWizardProps) => {
                     shouldCollapse
```
Comment (ntsim) [inline src/explore-education-statistics-common/src/modules/table-tool/components/LocationFiltersForm.tsx:170]: One more thing. This is poor design to couple `SummaryListItem` and `CollapsibleList` together. I would remove any of the collapsible nonsense out of `SummaryListItem` then just create `CollapsibleList`s as children i.e.

```typescript
<SummaryList>
  <SummaryListItem term="Whatever">
    <CollapsibleList>
      ...
    </CollapsibleList>
  <SummaryListItem>
</SummaryList>
```

This should probably always reveal other locations where we're using `div`s in `SummaryListItems`.

Tip: If you ever find yourself adding random props that have nothing to do with the component's concern just to add on little features like this, you can pretty much always achieve the same thing using just `children` or render props.

**Pull request merged**